lib/src/client.dart

```diff
@@ -120,7 +120,7 @@ class SockJSClient extends Disposable {
   /// Close this client.
   ///
   /// Optionally, a [closeCode] and [reason] can be provided.
-  void close([int closeCode, String reason]) {
+  void close([int? closeCode, String? reason]) {
     _jsClient.close(closeCode, reason);
   }
 
```

Comment:
Totally unrelated, but I noticed that below in `_onClose`, we have a few `// ignore`s that we could remove.